imagersite/imager_images/forms.py

```diff
@@ -35,3 +48,19 @@ def __init__(self, *args, **kwargs):
         self.fields['image'] = ImageField()
 
 
+class PhotoEditForm(ModelForm):
+    """
+    Form class for editing existing photo.
+    """
+    class Meta:
+        model = Photo
+        fields = ['title', 'description', 'date_published', 'published']
+
+    def __init__(self, *args, **kwargs):
+        kwargs.pop('username')
+        super().__init__(*args, **kwargs)
+        photo = kwargs['instance']
+        self.fields['title'].initial = photo.title
+        self.fields['description'].initial = photo.description
+        self.fields['date_published'].initial = photo.date_published
+        self.fields['published'].initial = photo.published
```

Comment:
You do not need to set these initial values. An edit form automatically gets filled in with data from the instance. You only need to do this for setting initial values of fields from a different model.